README.md

```diff
@@ -22,3 +22,5 @@ List of the [GitHub](https://github.com) usernames of all the people interested
 
 
 This list is not yet complete—we'll keep adding more people.
+
+Gitter link: https://gitter.im/py-sprints/pandas-mentoring
```

Comment:
Thanks @Aya-S 

I think the link won't be a link if it's not between `<` and `>`, like: `<link>`

Also, may be you can add couple of sentences on how to join it? It's also ok to do it in a follow up

Reply:
Hey @datapythonista, 
I made the change on the local branch I created to add the link and pushed the changes to my forked repo. the changes automatically showed here, which should be Okay this time :D. can you confirm that the PR looks fine to you?